substrate/primitives/runtime/src/traits/transaction_extension/mod.rs

```diff
@@ -605,14 +609,17 @@ impl<Call: Dispatchable> TransactionExtension<Call> for Tuple {
 impl<Call: Dispatchable> TransactionExtension<Call> for () {
 	const IDENTIFIER: &'static str = "UnitTransactionExtension";
 	type Implicit = ();
+	#[inline]
```

Comment:
`TransactionExtensionPipeline` take 32 generics that defaults to `()`, so I inlined methods here so hopefully the code gets optimized by the compiler.